engine/src/main/java/org/hibernate/search/query/dsl/impl/FacetRange.java

```diff
@@ -87,6 +87,20 @@ private String buildRangeString() {
 	}
 
 	@Override
+	public int hashCode() {
```

Comment:
No, it's not really. I only use it to have a string representation of the FacetRange. I'll go change it to something more understandable.

I also will fix the facetComparators duplicated code.